src/modules/rover_differential/module.yaml

```diff
@@ -113,6 +100,21 @@ parameters:
         decimal: 2
         default: 0.5
 
+      RD_MAX_DECEL:
+        description:
+          short: Maximum deceleration
+          long: |
+            Maximum decelaration is used to limit the deceleration of the rover.
```

Comment:
We need to be careful with what we refer to as "acceleration" and what with "deceleration". If you drive fast backwards and then try to get to a stop, is it then also using this "deceleration" limit? And do we really need asymmetric accelerating/decelerating settings?

Reply:
Deceleration is always about slowing down in absolute values (i.e. coming to a stop).
IMO the asymmetric parameters are necessary for the following reasons:
- The limit to stop a rover from doing a wheely is more strict than stopping it from doing a stoppie (rear wheel lifting up during deceleration).
- Also you don't want to limit the deceleration the same amount because you need to be able to come to a quick stop in an "emergency".